### ISSUE TITLE
How to know if a person has just placed his finger and is logged in?

### ISSUE THREAD

**Issue by degeta10** [dnaextrim/php_zklib#55]
I'm trying check if a person just logged-in in my device so I can pop an alert in my web.
Is it possible?

**Comment by send2dawood** [comment]
Yes possible .... by using cron job set a cron job which will check the device status every minutes
